datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/PrivateController.java

```diff
@@ -0,0 +1,15 @@
+package de.adorsys.datasafe.rest.impl.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/private")
+public class PrivateController {
```

Comment:
This will appear in swagger... maybe it is better to add it only for tests? As I understand it will be used in tests. For example, tests may declare additional ComponentScan

Reply:
removed. It was for personal test purposes during development